mappings/net/minecraft/item/Item.mapping

```diff
@@ -86,9 +86,9 @@ CLASS net/minecraft/class_124 net/minecraft/item/Item
 	FIELD field_460 RECORD_CAT Lnet/minecraft/class_124;
 	FIELD field_461 id I
 	FIELD field_462 maxCount I
-	FIELD field_463 texturePosition I
+	FIELD field_463 textureId I
 	FIELD field_464 handheld Z
-	FIELD field_465 fuel Z
+	FIELD field_465 hasSubtypes Z
 	FIELD field_466 craftingReturmItem Lnet/minecraft/class_124;
```

Comment:
There is a typo in the word "Return". This name was not changed in the PR but maybe it should be.

Reply:
fixed